just-install.json

```diff
@@ -737,6 +759,13 @@
         }
       }
     },
+    "virtualbox": {
+      "version": "5.0.4",
+      "installer": {
+        "kind": "custom",
```

Comment:
Cannot use the type "custom" as is, see other entries for an example of how to use "custom" installer. Additionally, are you sure that VirtualBox installation can't be covered by one of the recognised installer types?